internal/locate/region_request.go

```diff
@@ -1929,3 +1894,62 @@ func (s *baseReplicaSelector) backoffOnNoCandidate(bo *retry.Backoffer) error {
 	}
 	return bo.Backoff(args.cfg, args.err)
 }
+
+// failpointSendReqResult is used to process the failpoint For tikvStoreSendReqResult.
+func failpointSendReqResult(req *tikvrpc.Request, et tikvrpc.EndpointType) (
+	resp *tikvrpc.Response,
+	err error,
+) {
+	val, e := util.EvalFailpoint("tikvStoreSendReqResult")
+	if e != nil {
+		return
+	}
+	errMsg, ok := val.(string)
+	if !ok {
+		return
+	}
```

Comment:
It's ok to just follow the original pattern  `if val, err := util.EvalFailpoint("tikvStoreSendReqResult"); err == nil { ...`, instead of reformatting it. It's actually the expanded version of the `failpoint.Inject` pattern.